telegram/ext/jobqueue.py

```diff
@@ -21,10 +21,22 @@
 import logging
 import time
 import warnings
+import datetime
+from enum import Enum
 from threading import Thread, Lock, Event
 from queue import PriorityQueue, Empty
 
 
+class Days(Enum):
```

Comment:
And as far as I know, it would require too much code to be able to do an if statement to where sunday is day 0, as we don't even know how many locales have Sunday as the first day of the week.